hrms/patches/v14_0/update_title_in_employee_onboarding_template.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+
+
+def execute():
+	onboarding_templates = frappe.db.get_list(
+		"Employee Onboarding Template", fields=["name", "designation"]
+	)
+
+	for onboarding_template in onboarding_templates:
+		frappe.db.set_value(
+			"Employee Onboarding Template",
+			onboarding_template.get("name"),
+			{"title": onboarding_template.get("designation")},
+		)
```

Comment:
This for loop and 2 queries can be replaced with a simple `frappe.qb.update` statement

```python
Template = frappe.qb.DocType("Employee Onboarding Template")
(
	frappe.qb.update(Template)
	.set(Template.title, Template.designation)
	.where(Template.title.isnull())
).run()
```